### DISCUSSION TITLE
小米手机，分应用代理里找不到Google play 商店

### DISCUSSION THREAD

**Discussion by vicentory** [MetaCubeX/ClashMetaForAndroid#224] [Q&A] [answered]
小米手机，分应用代理里找不到Google play 商店，然后就无法上网，希望帮忙。
试了一下surfboard也是找不到play商店。

**Comment by jaydong2016** [comment]
同是小米，两个 app 都可以找到

**Reply by vicentory** [reply]
![Screenshot_2024-05-05-23-05-43-882_com.github.metacubex.clash.meta.jpg](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/10493597/a496b0b2-05be-42a2-858f-5ccd8ef03422)

搜索页面，的确看不到

**Reply by jaydong2016** [reply]
![Screenshot_2024-05-05-23-09-02-946_com.github.metacubex.clash.meta.jpg](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/23270314/f2a9ce8f-e690-473e-8c20-305233f6ee8c)

**Comment by potoo0** [comment, marked as answer]
右上角过滤里把 **系统** 取消勾选，此选项勾选时意思是不显示系统应用

**Reply by vicentory** [reply]
谢谢，可以了。
